.github/workflows/avm.platform.publish-tag.yml

```diff
@@ -48,7 +48,7 @@ jobs:
             Write-Output '::group::Publish tagged module to public bicep registry'
 
             # Load used functions
-            . (Join-Path $env:GITHUB_WORKSPACE 'avm' 'utilities' 'pipelines' 'platform' 'Publish-ModuleFromTagToPBR.ps1')
+            . (Join-Path $env:GITHUB_WORKSPACE 'avm' 'utilities' 'platform' 'Publish-ModuleFromTagToPBR.ps1')
```

Comment:
Solution under discussion

Reply:
Ping @ChrisSidebotham and @AlexanderSehr here for feedback. We have 2 platform PRs incoming

Reply:
ok - jap we still need to have that discussions / brainstorming ;)

Reply:
I am for this change as discussed offline, happy to have a quick trio sync if we can squeeze it in

Reply:
@eriqua - How about using an exclusion for the folder path within the original file structure?

e.g
` - !*/platform/*`